package.json

```diff
@@ -83,7 +83,7 @@
     "@bbc/psammead-locales": "^4.1.8",
     "@bbc/psammead-media-indicator": "^4.0.7",
     "@bbc/psammead-media-player": "^2.7.10",
-    "@bbc/psammead-most-read": "^4.1.2",
+    "@bbc/psammead-most-read": "^4.1.3",
```

Comment:
ooo, I'm actually in the progress of confirming the changes from this bump with UX 👀 I've got a separate PR covering this: https://github.com/bbc/simorgh/pull/6711

Reply:
You know I did mean to check if you were doing that, then forgot 😄 
I'll remove this one now

Reply:
Removed and snapshot updates removed